### ISSUE TITLE
Stopped working for me after 3 hours.

### ISSUE THREAD

**Issue by demirdegerli** [Sxvxgee/UnlimitedGPT#5]
I opened up the settings menu and it's probably a design change.

Browser opens up, driver closes the pop-up and stops working.

**Comment by Sxvxgee** [comment]
Did you update to the latest version? Run `pip install UnlimitedGPT -U` just to make sure and try again, if the problem persists, update me by commenting again and also attach your code.

**Comment by demirdegerli** [comment]
Already updated. Also the code:

```python
from UnlimitedGPT import ChatGPT

from gtts import gTTS 
from playsound import playsound 
import os

session_token = "yea"

api = ChatGPT(session_token)
api2 = ChatGPT(session_token)

def play(text):
    var = gTTS(text = text,lang = "en") 
    var.save("./temp.mp3")
    playsound("./temp.mp3")
    os.remove("./temp.mp3")

def send(msg):
    message = api.send_message(
    msg,
    input_mode="INSTANT"
    )
    print(message.response)
    play(message.response)
    send2(message.response)


def send2(msg):
    message = api2.send_message(
    msg,
    input_mode="INSTANT"
    )
    print(message.response)
    play(message.response)
    send(message.response)

send("What do you think about Barbies?")
```

**Comment by demirdegerli** [comment]
Also the UI looks different, can you take a look?

**Comment by demirdegerli** [comment]
Changing the textarea xpath in chatgpt_data.py worked. Also I made a PR for this: #6

**Comment by Sxvxgee** [comment]
Hey, sorry for the late reply, been busy with something. Thanks for attaching your code. As for the bug, I just tried the script as well and it seems you're right, they have update the website - yet again. Also, thanks for the PR, I'll review it soon. I just noticed some other functions are now broken because of the update, so I'll need to fix them as well and then I'll push an update to Github and PyPi. Should take less than an hour.
